pkg/operator/ceph/cluster/cluster.go

```diff
@@ -332,7 +332,14 @@ func (c *ClusterController) preClusterStartValidation(cluster *cluster, clusterO
 		cluster.Spec.Mon.Count = mon.DefaultMonCount
 	}
 	if cluster.Spec.Mon.Count%2 == 0 {
-		logger.Warningf("mon count is even (given: %d), should be uneven, continuing", cluster.Spec.Mon.Count)
+		return errors.Errorf("mon count %d cannot be even, must be odd to support a healthy quorum", cluster.Spec.Mon.Count)
+	}
+	if !cluster.Spec.Mon.AllowMultiplePerNode {
+		// Check that there are enough nodes to have a chance of starting the requested number of mons
+		nodes, err := c.context.Clientset.CoreV1().Nodes().List(metav1.ListOptions{})
+		if err == nil && len(nodes.Items) < cluster.Spec.Mon.Count {
```

Comment:
```suggestion
		if err == nil && len(nodes.Items) <= cluster.Spec.Mon.Count {
```

Shouldn't this be lower or equal than? Or were you thinking "enough nodes to run and failover"?

Reply:
The error is only when there are not enough nodes to run. If mon.count is 3 and there are 3 nodes, it's fine. If mon.count is 3 and there are 2 nodes it's an error.

Reply:
Oh yes somehow I read backward...